dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -332,272 +283,304 @@
 
     public static final String HEARTBEAT_TIMEOUT_KEY = "heartbeat.timeout";
 
-    public static final String CONNECT_TIMEOUT_KEY = "connect.timeout";
+    public static final String RECONNECT_KEY = "reconnect";
 
-    public static final String TIMEOUT_KEY = "timeout";
+    public static final int DEFAULT_RECONNECT_PERIOD = 2000;
 
-    public static final String RETRIES_KEY = "retries";
+    public static final String SEND_RECONNECT_KEY = "send.reconnect";
 
-    public static final String FAIL_BACK_TASKS_KEY = "failbacktasks";
+    public static final String CHECK_KEY = "check";
 
     public static final String PROMPT_KEY = "prompt";
 
     public static final String DEFAULT_PROMPT = "dubbo>";
+    // END dubbo-remoting-api
 
-    public static final String CODEC_KEY = "codec";
-
-    public static final String SERIALIZATION_KEY = "serialization";
-
-    public static final String EXTENSION_KEY = "extension";
-
-    public static final String KEEP_ALIVE_KEY = "keepalive";
-
-    public static final String OPTIMIZER_KEY = "optimizer";
+    // BEGIN dubbo-rpc-hessian
+    public static final String HESSIAN2_REQUEST_KEY = "hessian2.request";
 
-    public static final String EXCHANGER_KEY = "exchanger";
+    public static final boolean DEFAULT_HESSIAN2_REQUEST = false;
 
-    public static final String DISPACTHER_KEY = "dispacther";
+    public static final String HESSIAN_OVERLOAD_METHOD_KEY = "hessian.overload.method";
 
-    public static final String TRANSPORTER_KEY = "transporter";
+    public static final boolean DEFAULT_HESSIAN_OVERLOAD_METHOD = false;
 
-    public static final String SERVER_KEY = "server";
+    public static final String DEFAULT_HTTP_CLIENT = "jdk";
 
-    public static final String CLIENT_KEY = "client";
+    public static final String DEFAULT_HTTP_SERVER = "servlet";
 
-    public static final String ID_KEY = "id";
+    public static final String DEFAULT_HTTP_SERIALIZATION = "json";
```

Comment:
This is not used anymore, it is recommended to delete

Reply:
let's leave it as is, for now.